src/server_manager/install_scripts/do_install_server.sh

```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/env bash
```

Comment:
Let's revert this, it's a little risky thanks to the multitude of environments this script has to run in.